h2o-py/h2o/model/model_base.py

```diff
@@ -186,12 +186,17 @@ def predict_contributions(self, test_data):
         Note: Multinomial classification models are currently not supported.
 
         :param H2OFrame test_data: Data on which to calculate contributions.
+        :param Enum output_format: Specify how to output feature contributions in XGBoost - XGBoost by default outputs 
+            contributions for 1-hot encoded features, specifying a Compact output format will produce a per-feature
+            contribution. One of: ``"Original"``, ``"Compact"`` (default: ``"Compact"``).
```

Comment:
Says default is compact but in code it's original.

Reply:
Thank you for catching this. Fixed.